command/git.go

```diff
@@ -334,8 +336,42 @@ func (c *gitFetchProject) getPatchContents(ctx context.Context, comm client.Comm
 	return nil
 }
 
+// isMailboxPatch checks if the first line of a patch file
+// has "From ". If so, it's assumed to be a mailbox-style patch, otherwise
+// it's a diff
```

Comment:
Add link to https://git-scm.com/docs/git-am in comment (just so it's clear to later readers that mailbox is a git-am term).

Reply:
I think it's more appropriate to put this on the function just below it. I've already put it there.